deploy/helm/sumologic/conf/metrics/collector/otelcol/config.yaml

```diff
@@ -0,0 +1,123 @@
+            - source_labels:
+              - __meta_kubernetes_node_name
+              target_label: node
+            - source_labels:
+              - __meta_kubernetes_namespace
+              target_label: namespace
+            - source_labels:
+              - __meta_kubernetes_pod_name
+              target_label: pod
+            - source_labels:
+              - __meta_kubernetes_pod_container_name
+              target_label: container
+            - target_label: endpoint
+              replacement: https-metrics
+            - source_labels:
+              - __metrics_path__
+              target_label: metrics_path
+              action: replace
+            - source_labels:
+              - __address__
+              target_label: instance
+              action: replace
+        - job_name: cadvisor
+          scheme: https
+          authorization:
+            credentials_file: /var/run/secrets/kubernetes.io/serviceaccount/token
+          tls_config:
+            ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+            insecure_skip_verify: true
+          honor_labels: true
+          metrics_path: /metrics/cadvisor
+          kubernetes_sd_configs:
+            - role: node
+          metric_relabel_configs:
+            - action: replace
+              regex: .*
+              replacement: kubelet
+              source_labels: [__name__]
+              target_label: job
+            - action: keep
+              regex: (?:container_cpu_usage_seconds_total|container_memory_working_set_bytes|container_fs_usage_bytes|container_fs_limit_bytes|container_cpu_cfs_throttled_seconds_total|container_network_receive_bytes_total|container_network_transmit_bytes_total)
+              source_labels: [__name__]
+            ## Drop container metrics with container tag set to an empty string:
+            ## these are the pod aggregated container metrics which can be aggregated
+            ## in Sumo anyway. There's also some cgroup-specific time series we also
+            ## do not need.
+            - action: drop
+              source_labels: [__name__, container]
+              regex: (?:container_cpu_usage_seconds_total|container_memory_working_set_bytes|container_fs_usage_bytes|container_fs_limit_bytes);$
+            - action: labelmap
+              regex: container_name
+              replacement: container
+            - action: drop
+              source_labels: [container]
+              regex: POD
+            # TODO: The below can't be used due to https://github.com/open-telemetry/opentelemetry-operator/issues/958
+            # - action: labeldrop
+            #   regex: (id|name)
+          relabel_configs: *relabel_configs  # partially copied from what operator generates
+    target_allocator:
+      endpoint: http://{{ template "sumologic.metadata.name.metrics.targetallocator.name" . }}
+      interval: 30s
+      collector_id: ${POD_NAME}
+extensions:
```

Comment:
We should perhaps extract the extension definitions to another file, to prevent duplication. This can be done in another PR.